### PR TITLE
USWDS - Utilities: Update inaccurate comments

### LINKED ISSUE #5860: USWDS - Bug: Inaccurate utility code comments
From @aduth in PR #5700:
> As a developer interested in optimizing the output size of bundles, I would like to eliminate utility classes not in use by configuring the $output-these-utilities theme setting. This is already difficult enough as-is since there's not an easy reference of utility name to the associated output CSS classes, but it's made worse by inaccuracies within code documentation for the utility class implementations.
> 
> Edit: In fairness, there is a list within the [v3.0 migration guide](https://designsystem.digital.gov/documentation/migration/#managing-utility-classes), but it's not easy to find, and my instinct was to trust the source comments as a reference.

### PR DESCRIPTION
**Pull request by amyleadem**
# Summary

**Updated the code comments on utility Sass partials.** These comments now reflect the correct utility class names and values.

> [!Important]
> This PR builds on the changes added by @aduth in PR #5700. Thank you @aduth!

## Breaking change

This is not a breaking change. This update only affects code comments.

## Related issue

Closes #5860 

## Related pull requests

This PR builds on the changes from @aduth in PR #5700.

## Preview link

N/A

## Problem statement

The code comments on the files in the `packages/uswds-utilities/src/styles/rules/` directory are outdated and inaccurate. These code comments should be updated so that developers can use the code comments as reference (especially for utilities not documented on our website).

## Solution

This PR updates outdate utility class references to match uswds 3.0 naming conventions. 

## Testing and review
1. Confirm that all files in the `packages/uswds-utilities/src/styles/rules/` have comments that reflect current USWDS naming structures
1. Confirm comments are formatted consistently

### CODE REVIEW

File: packages/uswds-utilities/src/styles/rules/add-aspect.scss [hunk 1]
```diff
@@ -3,16 +3,17 @@
...
-  padding-bottom: 100%;
+  aspect-ratio: [value];
 ----------------------------------------
 example:
-  .aspect-16x9
+  .add-aspect-16x9 {
+    height: inherit;
+    padding: inherit;
+    aspect-ratio: 1.7777777778;
+    max-width: 100%; }
```
Comment (mejiaj) [inline packages/uswds-utilities/src/styles/rules/add-aspect.scss:16]: Love the improved comments.

Small change to indent to match standards below.

```suggestion
    max-width: 100%; 
  }
```

File: packages/uswds-utilities/src/styles/rules/add-list-reset.scss
Comment (mejiaj) [inline packages/uswds-utilities/src/styles/rules/add-list-reset.scss:1]: > [!TIP]
> Not alphabetical order, but it's output in `uswds.css`.

```css
/* uswds.css */
.add-list-reset{
  margin-bottom:0;
  margin-top:0;
  padding-left:0;
  list-style:none;
}
```

File: packages/uswds-utilities/src/styles/rules/background-color.scss [hunk 2]
```diff
@@ -9,8 +9,8 @@ output:
   background-color: [value];
 ----------------------------------------
 example:
-  .bg-red-50v {
-    background-color: #be4900; }
+  .bg-red {
+    background-color: #e52207; }
```
Comment (mejiaj) [inline packages/uswds-utilities/src/styles/rules/background-color.scss:13]:
```suggestion
    background-color: #e52207; 
  }
```

File: packages/uswds-utilities/src/styles/rules/border-color.scss [hunk 3]
```diff
@@ -1,16 +1,16 @@
...
+  .border-[key]
 ----------------------------------------
 output:
-  border[-color]: [color];
+  border-color: [value];
 ----------------------------------------
 example:
   .border-red {
-    border-color: #be4900; }
+    border-color: #e52207; }
```
Comment (mejiaj) [inline packages/uswds-utilities/src/styles/rules/border-color.scss:13]:
```suggestion
    border-color: #e52207; 
  }
```

File: packages/uswds-utilities/src/styles/rules/border-radius.scss [hunk 4]
```diff
@@ -3,17 +3,17 @@
...
 ----------------------------------------
 example:
-  .border-radius-0 {
+  .radius-0 {
     border-radius: 0; }
 
-  .border-radius-pill {
-    border-radius: 10em; }
+  .radius-pill {
+    border-radius: 99rem; }
```
Comment (mejiaj) [inline packages/uswds-utilities/src/styles/rules/border-radius.scss:16]:
```suggestion
    border-radius: 99rem; 
  }
```

File: packages/uswds-utilities/src/styles/rules/border.scss [hunk 5]
```diff
@@ -12,11 +12,11 @@ output:
   border-[modifier]: [value];
 ----------------------------------------
 example:
-  .border-t-1px {
+  .border-top-1px {
     border-top: 1px solid; }
 
   .border-0 {
-    border: none; }
+    border: 0 solid; }
```
Comment (mejiaj) [inline packages/uswds-utilities/src/styles/rules/border.scss:19]:
```suggestion
    border: 0 solid; 
  }
```

File: packages/uswds-utilities/src/styles/rules/border.scss [hunk 6]
```diff
@@ -12,11 +12,11 @@ output:
   border-[modifier]: [value];
 ----------------------------------------
 example:
-  .border-t-1px {
+  .border-top-1px {
     border-top: 1px solid; }
```
Comment (mejiaj) [inline packages/uswds-utilities/src/styles/rules/border.scss:16]:
```suggestion
    border-top: 1px solid; 
  }
```

File: packages/uswds-utilities/src/styles/rules/overflow.scss [hunk 7]
```diff
@@ -3,10 +3,10 @@
 overflow
 ----------------------------------------
 usage:
-  .overflow-[modifier]
+  .overflow-[modifier]*-[key]
```
Comment (mejiaj) [inline packages/uswds-utilities/src/styles/rules/overflow.scss:6]: What's the value of the `*`?

**Review by mejiaj — CHANGES_REQUESTED** [review]
Thanks for making these fixes. I was able to test comments are accurate and up-to-date. 

Can we fix the closing `}` in the code examples?

File: packages/uswds-utilities/src/styles/rules/opacity.scss [hunk 8]
```diff
@@ -11,6 +11,9 @@ output:
 example:
   .opacity-0 {
     opacity: 0; }
+
+  .opacity-30 {
+    opacity:0.3; }
```
Comment (mahoneycm) [inline packages/uswds-utilities/src/styles/rules/opacity.scss:16]: Missing a space 
```suggestion
  .opacity-30 {
    opacity: 0.3; }
```

**Review by mahoneycm — CHANGES_REQUESTED** [review]
This is looking great, thank you for combing through these and updating.

Besides James' new line comment, I noticed a space missing for one value. Minuscule change but might as well while we're here 😄

File: packages/uswds-utilities/src/styles/rules/overflow.scss (hunk 7 above)
Comment (amyleadem) [reply packages/uswds-utilities/src/styles/rules/overflow.scss:6]: I _believe_ it indicates that the modifier is optional. At least, that is my assumption based on the other comments with this format. I read it as the class can can be written as `.overflow-hidden` or `.overflow-y-hidden`.

File: packages/uswds-utilities/src/styles/rules/add-aspect.scss (hunk 1 above)
Comment (amyleadem) [reply packages/uswds-utilities/src/styles/rules/add-aspect.scss]: This was updated in [267fc6d](https://github.com/uswds/uswds/pull/5859/commits/267fc6d3e58a1be2aaff128aeca4f8e6fad024b8)

File: packages/uswds-utilities/src/styles/rules/background-color.scss (hunk 2 above)
Comment (amyleadem) [reply packages/uswds-utilities/src/styles/rules/background-color.scss]: This was updated in [267fc6d](https://github.com/uswds/uswds/pull/5859/commits/267fc6d3e58a1be2aaff128aeca4f8e6fad024b8)

File: packages/uswds-utilities/src/styles/rules/border-radius.scss (hunk 4 above)
Comment (amyleadem) [reply packages/uswds-utilities/src/styles/rules/border-radius.scss]: This was updated in [267fc6d](https://github.com/uswds/uswds/pull/5859/commits/267fc6d3e58a1be2aaff128aeca4f8e6fad024b8)

File: packages/uswds-utilities/src/styles/rules/border.scss (hunk 5 above)
Comment (amyleadem) [reply packages/uswds-utilities/src/styles/rules/border.scss]: This was updated in [267fc6d](https://github.com/uswds/uswds/pull/5859/commits/267fc6d3e58a1be2aaff128aeca4f8e6fad024b8)

File: packages/uswds-utilities/src/styles/rules/border-color.scss (hunk 3 above)
Comment (amyleadem) [reply packages/uswds-utilities/src/styles/rules/border-color.scss]: This was updated in [267fc6d](https://github.com/uswds/uswds/pull/5859/commits/267fc6d3e58a1be2aaff128aeca4f8e6fad024b8)

File: packages/uswds-utilities/src/styles/rules/opacity.scss (hunk 8 above)
Comment (amyleadem) [reply packages/uswds-utilities/src/styles/rules/opacity.scss]: This was updated in [267fc6d](https://github.com/uswds/uswds/pull/5859/commits/267fc6d3e58a1be2aaff128aeca4f8e6fad024b8)

**Comment by amyleadem** [comment]
@mejiaj and @mahoneycm  Thanks for those comments. I've moved the closing braces in the comments to a new line to meet standards and added the missing space to the opacity comment. Please let me know if I missed anything!

**Review by mejiaj — APPROVED** [review]

**Review by mahoneycm — APPROVED** [review]

**Review by thisisdano — APPROVED** [review]
Thanks @aduth and @amyleadem! A somewhat thankless but important task

**Comment by thisisdano** [comment]
But now it's thankless no longer :)

**Pull request merged**